autocorpus/autocorpus.py

```diff
@@ -304,17 +377,60 @@ def process_files(self):
         if "documents" in self.tables and not self.tables["documents"] == []:
             self.has_tables = True
 
+    def process_files(
```

Comment:
None of these new methods need to be in the `Autocorpus` class, and I think they actually should be in a separate module specifically for processing supplementary files.

But I think I will tackle this in #185